docker-compose.yml

```diff
@@ -18,3 +18,6 @@ services:
       - xpack.security.enabled=false
     ports:
       - "9200:9200"
+  toxiproxy:
+    image: shopify/toxiproxy:2.1.4
```

Comment:
Not sure if there is a compability requirement by if we are updating this, we should update gem to 2.x as well.

Reply:
I can put any version in there. Anyway we're ok AFAIK.

from https://github.com/Shopify/toxiproxy-ruby#toxiproxy-ruby
> toxiproxy-ruby 1.x (latest) is compatible with the Toxiproxy 2.x series. toxiproxy-ruby 0.x is compatible with the Toxiproxy 1.x series.

We're using 1.0.3 gem version. Info about Gem version is at https://github.com/Shopify/toxiproxy-ruby/pull/37. Should I bump the gem as well to keep dependencies fresh?

Reply:
Yeah tests are passing so we are fine. I would update gem as well anyway.